.github/workflows/lint-pr.yml

```
@@ -0,0 +1,19 @@
+name: lint-pr
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  validate:
+    name: validate-pull-request-title
+    runs-on: ubuntu-22.04
```

Why not this?

```suggestion
    runs-on: ubuntu-latest
```

I think it's a copy paste. I don't think this would require such old LTS release